share/doc/wake/logging.md

```diff
@@ -0,0 +1,64 @@
+# Logging
+
+This article gives a high level overview of how wake logging works.
+Check out [print.wake](https://github.com/sifive/wake/blob/master/share/wake/lib/core/print.wake)
+for commented code.
+
+## How Logging works on Wake Execution commands
+
+Every `Job` will emit a stderr and a stdout.
```

Comment:
Nit 1: stderr and stdout are file descriptors which just exist for pretty much all Unix processes. You probably mean "emit to" rather than "emit", but it might be clearer to just say "print to".

Nit 2: Not all Jobs will necessarily write to either stderr or stdout; it depends on whether the job itself tries to print to either of those file descriptors.

Here's my version of the sentence I think you wanted:

```suggestion
Every `Job` run by wake will have its own stdout and stderr, and when the underlying process prints to stdout or stderr, it will go the `Job`-specific stdout and stderr, not the stdout and stderr of the `wake` process itself.
```